sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -774,6 +774,13 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val AUTO_UPDATE_SIZE =
+    buildConf("spark.sql.statistics.autoUpdate.size")
+      .doc("Enables automatic update for table size once table's data is changed. " +
+      "Note that this update also removes other statistics (e.g. row count and column statistics)")
```

Comment:
this is misleading, even they don't do auto update, row count and column statistics will also be removed.